Runtime/Tx/RaycastLiDARTx.cs

```diff
@@ -29,18 +29,18 @@ protected override void OnStart()
                 data.fields[i] = new PointField();
                 data.fields[i].name = ((char)('x' + i)).ToString();
                 data.fields[i].offset = (uint)(4 * i);
```

Comment:
offset should be calculated based on previous field size

for example look at this:
```
fields:
- name: x
  offset: 0
  datatype: 7
  count: 1
- name: y
  offset: 4
  datatype: 7
  count: 1
- name: z
  offset: 8
  datatype: 7
  count: 1
- name: intensity
  offset: 12
  datatype: 7
  count: 1
```